src/target/jtag_devs.c

```diff
@@ -57,5 +61,6 @@ jtag_dev_descr_t dev_descr[] = {
 	{.idcode = 0x20270013, .idmask = 0xFFFFFFFF, .descr = "Intel: i80386ex."},
 	{.idcode = 0x07B7617F, .idmask = 0xFFFFFFFF, .descr = "Broadcom: BCM2835."},
 	{.idcode = 0x4BA00477, .idmask = 0xFFFFFFFF, .descr = "Broadcom: BCM2836."},
+	{.idcode = 0x790007A3 , .idmask = 0xFFFFFFFF, .descr = "GigaDevice: GD32VF103CBT6 (BSD)."},
```

Comment:
Is there actually a second GD32VF idcode?, or was this perhaps a copy & paste mistake?